backend/app/api/ws/device_namespace.py

```
@@ -205,14 +208,46 @@ def _register_device(
+    from app.models.kind import Kind
+
+    with get_db_session() as db:
+        device = (
+            db.query(Kind)
+            .filter(
+                and_(
+                    Kind.user_id == user_id,
+                    Kind.kind == "Device",
+                    Kind.namespace == "default",
+                    Kind.name == device_id,
+                    Kind.is_active == True,
```

Ruff (0.15.17)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)